modules/testing/pages/integration/standardized-outputs.adoc

```diff
@@ -0,0 +1,21 @@
+= Standardized outputs
+
+Konflux is using standardized outputs in various places, this part of docs will provide more detail information about it.
+Each one of the results described below is rendered in the UI, you simply click on task name and panel with details is showed
+containing all results from the task. 
+
+== TEST_OUTPUT
+
+Each test in build pipelinerun provides TEST_OUTPUT result [json] that sumarizes its outcome. It's then being evaluated by Conforma team whether the product can be released or not.
+
+This result is also parsed by Integration-service, that means you can use it to fail individual test tasks without causing the entire pipeline to fail.
```

Comment:
Maybe we can expand on this, something like:
```suggestion
This result type can be also used in integration test pipelines and tasks where it can be parsed by the integration service when deciding the outcome of the integration test. This means you can use it to fail individual test tasks by emitting this result instead of failing the task, thus failing and stopping the entire pipeline. With this approach it is possible to run multiple tests within the same pipeline and pass or fail them individually. 
```